### PR TITLE
fix: Correct demand Icon spacing

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses an issue in [CX-3077](https://artsyproduct.atlassian.net/browse/CX-3077)

## Description

Correct demand Icon spacing ([more context](https://github.com/artsy/force/pull/11344#discussion_r1021939091)).

### CODE REVIEW

File: src/Components/Artwork/Details.tsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ const HighDemandInfo = () => {
   return (
     <Flex flexDirection="row" alignItems="center">
       <HighDemandIcon width={16} height={16} />
-      <Text variant="xs" color="blue100" ml={0.3}>
+      <Text variant="xs" color="blue100" ml="3px">
```
Comment (dzucconi) [inline src/Components/Artwork/Details.tsx:142]: I guess but it sounds like you just want a normal `" "` or something

Comment (olerichter00) [reply src/Components/Artwork/Details.tsx:142]: Hmm, placing a `{" "}` behind`<HighDemandIcon width={16} height={16} />` or inside of `Text`  doesn't seem to work 🤔

Comment (dzucconi) [reply src/Components/Artwork/Details.tsx:142]: Inside of `Text` `&nbsp;` would work

**Comment by dzucconi** [comment]
You'll need to rebase with `main` for acceptance to pass.

File: src/Components/Artwork/Details.tsx (hunk 1 above)
Comment (olerichter00) [reply src/Components/Artwork/Details.tsx]: It works 👍

**Pull request closed without merging**